README.md

```diff
@@ -1 +1,51 @@
-# linkerd-inject
+# Transparent proxy inection
+
+We can do transparent proxying of requests to linkerd via `iptables` rules.
+
+This script injects an `initContainer` into the user's k8s config. This
+`initContainer` sets up `iptables` rules for tranparent proxying of requests to a
+[Daemonset linkerd](https://github.com/linkerd/linkerd-examples/blob/master/k8s-daemonset/k8s/linkerd.yml).
+
+It injects the following initContainer (which you could add to your config
+manually if you would rather not use the script). The script uses the
+`pod.beta.kubernetes.io/init-containers` annotation, which you would need to use
+if you are running a version of the Kubernetes Apiserver before 1.6.
+
+```
+initContainers:
+- name: init
+  image: buoyantio/istio-init:v1
+  env:
+  - name: NODE_NAME
+    valueFrom:
+      fieldRef:
+        fieldPath: spec.nodeName
+  args:
+    - -p
+    - "4140" # port of the Daemonset linkerd's incoming router
+  imagePullPolicy: Always
+  securityContext:
+    capabilities:
+      add:
+      - NET_ADMIN
+```
+
+It is based on Istio's method of
+[injecting sidecars](https://github.com/istio/pilot/blob/master/doc/proxy-injection.md),
```

Comment:
Super minor, but since this code is in flux right now, maybe consider pinning this to a specific release tag, rather than master. How about `https://github.com/istio/pilot/blob/stable-a632657/doc/proxy-injection.md`?